### ISSUE TITLE
Change in require behaviour with latest 5.4.0 version.

### ISSUE THREAD

**Issue by jkogara** [getsentry/sentry-ruby#1856]
### Issue Description

I have an in house gem that wraps the sentry library. Before the 5.4.0 release 

```ruby
require 'sentry-ruby'
```

loaded the sentry library, this now raises the following exception 

```shell
LoadError:
  cannot load such file -- sentry-ruby
```

### Reproduction Steps

Add `sentry-ruby` as a runtime dependency to a newly created gem.
Add `require 'sentry-ruby'` to the entry point to the gem
run `./bin/console` 

```shell
[ |main S:15 U:2 ?:1 ✗|]» ./bin/console 
Traceback (most recent call last):
	3: from ./bin/console:5:in `<main>'
	2: from ./bin/console:5:in `require'
	1: from /home/jkogara/src/hopin/tesy-sentry/lib/tesy/sentry.rb:4:in `<top (required)>'
/home/jkogara/src/hopin/tesy-sentry/lib/tesy/sentry.rb:4:in `require': cannot load such file -- sentry-ruby (LoadError)
```

### Expected Behavior

It should load without exception

### Actual Behavior

There is an exception

### Ruby Version

2.7.2p137

### SDK Version

5.4.0

**Comment by sl0thentr0py** [comment]
fixing asap, thx for reporting!

**Comment by sl0thentr0py** [comment]
@jkogara released 5.4.1, sorry about that!
